libraries/AP_Mount/AP_Mount_R10C.h

```diff
@@ -15,12 +15,18 @@
 #include <RC_Channel_aux.h>
 #include "AP_Mount_Backend.h"
 
+enum r10c_gimbal_state_t {
+    R10C_GIMBAL_STATE_NOT_PRESENT = 0,
+    R10C_GIMBAL_STATE_PRESENT_RUNNING
```

Comment:
More for my curiosity: is there a reason to not explicitly define R10C_GIMBAL_STATE_PRESENT_RUNNING = 1?